src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -37,9 +37,12 @@ public DefaultServer(List<Type> typesToInclude)
             configuration.EnableInstallers();
 
             configuration.DisableFeature<TimeoutManager>();
-            configuration.Recoverability()
-                .Delayed(delayed => delayed.NumberOfRetries(0))
-                .Immediate(immediate => immediate.NumberOfRetries(0));
+            if (!endpointConfiguration.PerformDefaultRetries)
```

Comment:
do we really need this? The default behavior for tests is to disable all retries. If you want retries, you probably have a good idea of what retry config you need and the default values will most likely be the wrong configuration anyway.